src/components/Observations/ObservationsDataView.tsx

```diff
@@ -1,14 +1,16 @@
 import { useAppSelector } from 'src/redux/store';
-import { selectPlantingSiteObservationsResults } from 'src/redux/features/observations/observationsSelectors';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+import { selectMergedPlantingSiteObservations } from 'src/redux/features/observations/observationsSelectors';
 import ListMapView from 'src/components/ListMapView';
 
 export type ObservationsDataViewProps = {
   selectedPlantingSiteId: number;
 };
 
 export default function ObservationsDataView({ selectedPlantingSiteId }: ObservationsDataViewProps): JSX.Element {
+  const defaultTimeZone = useDefaultTimeZone();
   const observationsResults = useAppSelector((state) =>
```

Comment:
@charlie-tf this is the component that selects the merged observations data, which can in turn be fed into the map component on line 21